sphinxgallery/docs_resolv.py

```diff
@@ -409,6 +409,8 @@ def embed_code_links(app, exception):
     """Embed hyperlinks to documentation into example code"""
     if exception is not None:
         return
+    if not app.builder.config.plot_gallery:
+        return  # no need to embed
```

Comment:
I missed that in my previous reviews. Why don't you want to put hyperlinks inside the example in case plot_gallery is False? Is it because it can fail if there is a parsing error of the script ?